easybuild/easyconfigs/p/pIRS/pIRS-2.0.2_fix-Makefiles.patch

```diff
@@ -0,0 +1,130 @@
+Fix hardcoded stuff in Makefiles
+
+author: Pavel Grochal (INUITS)
+diff -ru --new-file pirs-2.0.2.orig/Makefile pirs-2.0.2/Makefile
+--- pirs-2.0.2.orig/Makefile	2016-08-14 07:10:03.000000000 +0200
++++ pirs-2.0.2/Makefile	2020-04-28 10:34:20.369344282 +0200
+@@ -1,3 +1,10 @@
++CC ?= gcc
++CXX ?= g++
++CFLAGS ?= -g -O2
++CXXFLAGS ?= -g -O2
++PKGDATADIR ?= "/usr/local/share/pirs"
++export
++
+ all: simulator stator symlinks
+ 
+ PERL_LIST = stator/baseCallingMatrix/baseCalling_Matrix_analyzer \
+@@ -49,4 +56,3 @@
+ 	git push github master
+ 	git push
+ 	git push google master
+-
+diff -ru --new-file pirs-2.0.2.orig/src/pirs/gccMakefile pirs-2.0.2/src/pirs/gccMakefile
+--- pirs-2.0.2.orig/src/pirs/gccMakefile	2016-08-14 07:10:03.000000000 +0200
++++ pirs-2.0.2/src/pirs/gccMakefile	2020-04-28 11:43:36.611256253 +0200
+@@ -1,52 +1,23 @@
+-CFLAGS = -pipe -mtune=generic -msse2 -fopenmp -DSFMT_MEXP=19937 -DHAVE_CONFIG_H -DPKGDATADIR='"/usr/local/share/pirs"' -I. -I.. -ISFMT-src-1.4
+-
+-ifeq ($(shell uname -s),Darwin)
+-CFLAGS += -march=core2 -Bstatic -L/opt/local/lib -I/opt/local/include
+-else
+-CFLAGS += -static
+-endif
+-
+-#GCC_VERSION := `gcc -dumpversion`
+-GCC_VERSION := $(shell gcc -dumpversion)
+-GCC_MAJOR := $(shell echo $(GCC_VERSION) | cut -d'.' -f1)
+-#GCC_MINOR := $(shell echo $(GCC_VERSION) | cut -d'.' -f2)
+-#GCC_SUB := $(shell echo $(GCC_VERSION) | cut -d'.' -f3)
+-GCC4_VER := $(shell echo $(GCC_VERSION) | cut -d'.' -f2,3)
+-
+-CPPFLAGS += $(CFLAGS)
+-
+-ifeq ($(GCC_MAJOR),4)
+-ifeq ($(shell expr 2.4 \<= $(GCC4_VER)),1)
+-CFLAGS += -march=native
+-CPPFLAGS += -march=native
+-endif
+-ifeq ($(shell expr 3.6 \<= $(GCC4_VER)),1)
+-#CPPFLAGS += -std=c++0x
+-endif
+-endif
+-
+-LDLIB     = -lz -lpthread
+-LDFLAGS   = -Wl,-O1 -Wl,--sort-common -Wl,--enable-new-dtags -Wl,--hash-style=both $(LDLIB)
+ 
+ SRCDIRS = . SFMT-src-1.4
+ SRCEXTS = .C .cc .cpp .CPP .c++ .cxx .c
+-OLevel= -O2
++
+ WARNFLAGS = -Wfloat-equal -Wall
+ SOURCES = $(foreach d,$(SRCDIRS),$(wildcard $(addprefix $(d)/*,$(SRCEXTS))))
+ 
+-# g++ -DHAVE_CONFIG_H -I. -DSFMT_MEXP=19937 -DPKGDATADIR='"/usr/local/share/pirs"' -Wall -Wextra -msse2  -fopenmp -g -O2 *.cpp SFMT-src-1.4/SFMT.c -lz
++EXTRAFLAGS = -lz -lpthread -pipe -mtune=generic -msse2 -fopenmp -DSFMT_MEXP=19937 -DHAVE_CONFIG_H  -I. -I.. -ISFMT-src-1.4
+ 
+ all: pirs
+ 	@echo Souce Files:[$(SOURCES)]
+ 
+ pirs: $(SOURCES)
+-	g++ $(WARNFLAGS) $(CPPFLAGS) $(OLevel) -o $@ $^ $(LDFLAGS)
++	$(CXX) $(WARNFLAGS) $(CPPFLAGS) -o $@ $^ $(LDFLAGS) $(EXTRAFLAGS) -DPKGDATADIR='"$(PKGDATADIR)"'
```

Comment:
Should use `$(CXXFLAGS)` rather than `CPPFLAGS`?

Any reason not to include `-DPKGDATADIR='"$(PKGDATADIR)"'` in `EXTRAFLAGS` above?

(same below)